app/src/main/java/ru/yandex/practicum/contacts/presentation/filter/model/FilterContactTypeUi.java

```diff
@@ -37,4 +41,8 @@ public int hashCode() {
         result = 31 * result + (selected ? 1 : 0);
         return result;
     }
+    @Override
+    public boolean theSameAs(FilterContactTypeUi other) {
+        return this == other;
```

Comment:
В этом методе нужно поверхностно сравнить объекты, хорошим полем здесь будет `contactType`:
```suggestion
        return this.getContactType() == other.getContactType();
```